index.html

```diff
@@ -80,121 +86,88 @@ <h1>I’m Beyk
 					</div>
 				</div>
 			</section>
-			<!--Start work section-->
-			<section class="work" id="work">
-				<div class="work-tonic">
-					<div class="snapshoot-div">
-						<img src="images/Snapshoot1.png" alt="Project image" class="snapshoot"/>
-						<img src="images/snapshoot1-des.png" alt="Project image" class="desk-snapshoot"/>
-					</div>
-					<div class="work-info">
-						<h2>Tonik</h2>
-						<div class="text-info">
-							<ul>
-								<li class="client-name">CANOPY</li>
-								<li class="role">Back End Dev</li>
-								<li class="year">2015</li>
-							</ul>
-						</div>
-						<p class="work-text"
-							>A daily selection of privately personalized reads; no accounts or sign-ups required.</p
-						>
-						
-							<ul class="tags">
-								<li>html</li>
-								<li>css</li>
-								<li>javaScript</li>
-							   </ul>
-						
-						<button class="see-proj">See project</button>
-					</div>
-				</div>
 			
-				
-				<div class="work-tonic reverse">
-					<div class="snapshoot-div">
-						<img src="images/Snapshoot2.png" alt="Project image" class="snapshoot"/>
-						<img src="images/snapshoot2-des.png" alt="Project image" class="desk-snapshoot"/>
+			<section class="detail-section">
```

Comment:
- [ ] Please kindly correct the margins, padding, line-heights, and alignments of the modal elements, so the modal matches the Figma more closely (both views).
- [ ] Please, let's add the circles that go between "canopy", "back end dev", and "2015" (both views).
- [ ] Please kindly fix the font styles (font-weight, font-size, line-height, etc) on the modals, some are not matching the Figma.  One example is the description text.  Another example is the "Tonic" heading. (both views)

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/152496663-1eab9e63-374e-4bb0-9926-a9a284f9f8ff.png) | ![image](https://user-images.githubusercontent.com/57421823/152496774-b2de3c28-f5d4-4bd1-9247-bf185bd6bd8e.png)

- [ ] Please, kindly center these buttons on the mobile view, like shown on the template. (mobile view).
- [ ] Also, let's correct the font styling for the text inside said buttons (both views).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/152497609-905e0e25-6268-473e-a495-e0923be23dc4.png) | ![image](https://user-images.githubusercontent.com/57421823/152497663-2523efcc-ab06-4428-b886-f505695e7eac.png)

That way, your pop-ups will look strikingly beautiful! 💪